pyproject.toml

```diff
@@ -51,25 +51,21 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">=3.7.1,<3.11"
-Jinja2  = ">=2.11.3"
+Jinja2 = "*"  # mlflow-skinny -> packaging -> pyparsing uses it, but doesn't require it
 aiohttp  = ">=3.7.3,<3.8.0"
 boto3  = ">=1.16.24"
 cloudpickle  = ">=2.0.0"
 cryptography  = ">=3.4.7"
 humanize  = ">=3.11.0"
-idna = "<3"
 mlflow-skinny = ">=1.25.0"
 scipy = "*"  # needed for mlflow.sklearn
 networkx  = ">=2.5"
 pandas = "1.3.5"
-pickle5 = { version = "~0.0.11", python = "<3.8" }
 polling  = ">=0.3.1"
-protobuf  = ">=3.12.0"
```

Comment:
Already comes from `layer-api` -> `grpc-tools`, no need to constraint further here.